modules/dynamic-plugin-localization.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/overview-dynamic-plugin.adoc
+
+:_mod-docs-content-type: Procedure
+[id="dynamic-plugin-api_{context}"]
+= Translating messages with react-i18next
```

Comment:
Hey @spadgett I noticed I missed some things on i18n procedure. I was trying to make it more of a procedure than how it is in the README. Can you let me know if this is the correct order and if these are accurate. Thanks! 😄